src/encoded/static/components/navigation.js

```diff
@@ -301,13 +301,17 @@ export const Breadcrumbs = (props) => {
                     accretingTip += crumb.tip ? (accretingTip.length ? ' and ' : '') + crumb.tip : '';
                 }
 
+                // Append released tag to link and link title if the object is released itself
+                const releasedLink = (props.crumbsReleased) ? '&status=released' : '';
```

Comment:
Have to add `crumbsReleased` to propTypes: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md